conbench/entities/hardware.py

```diff
@@ -188,9 +188,27 @@ class MachineCreate(marshmallow.Schema):
 
 
 class ClusterCreate(marshmallow.Schema):
-    name = marshmallow.fields.String(required=True)
-    info = marshmallow.fields.Dict(required=True)
-    optional_info = marshmallow.fields.Dict(required=True)
+    name = marshmallow.fields.String(
```

Comment:
I believe you've got the description for info here (and then the description for name down below, yeah?

```suggestion
    info = marshmallow.fields.String(
```

Reply:
Thanks for the pointer, I have swapped the entries